.template-lintrc.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
+  extends: 'octane',
+  rules: {
+    'no-bare-strings': true
```

Comment:
Without specifying the rule, the CLI displays a warning message.

Reply:
Ahh, righto (this is because we have ember-intl).